addons/web/static/src/js/core/rpc.js

```diff
@@ -14,7 +14,7 @@ return {
      * @returns {Promise<any>}
      */
     query: function (params, options) {
-        var query = this.buildQuery(params);
+        var query = rpc.buildQuery(params);
```

Comment:
Yes, by now using `this.env.services.rpc` (now that it uses MockServer, we can use it!), it's implementation detail in `web.test_env` is to call `buildQuery()` in this file as a standalone. This is a problem, because this file calls other functions by means of `this`, and `this` is `undefined`... This fix is to avoid crash.

I think our tests show this issue because we have some tests that use `search_read` or `read` and rely on defaults in MockServer, and these kinds of `rpc` reproduce the crash I had that motivates making fixes in this file.